src/faq.md

```diff
@@ -266,6 +280,21 @@ You can also use the [Flutter framework]({{site.flutter}}), which has [web suppo
 [No.]({{site.news}}/2015/03/dart-for-entire-web.html)
 Dart is designed to compile to JavaScript to run across the modern web.
 
+### Q. Is Dart single-threaded on the web?
+
+Dart is *mostly* single-threaded on the web. 
+Web apps can use concurrency features like async-await,
+futures, and streams, but not isolates.
```

Comment:
Async-await are not _concurrency_ features.

Maybe just:

> Dart is *mostly* single-threaded on the web, since web apps cannot use isolates.
> Web apps can instead use [web workers] to run code concurrently,
> but web workers have different capabilities and restrictions than isolates,
> and are not as easy or efficient to use. For more information, 
> ...

Reply:
**suggestion:** Simplify again.


> Dart web apps cannot use isolates. To run code concurrently,
> Web apps use [web workers]. 
> Web workers have different capabilities and restrictions than isolates.
> They lack the ease and efficiency of isolates. To learn more, 
> ...

Reply:
Better!